src/libsyncengine/update_detection/update_detector/updatetreeworker.cpp

```diff
@@ -379,6 +386,11 @@ ExitCode UpdateTreeWorker::step4DeleteFile() {
             // Transform a Delete and a Create operations into one Edit operation.
             // Some software, such as Excel, keeps the current version into a temporary directory and move it to the destination,
             // replacing the original file. However, this behavior should be applied only on local side.
+            SyncPath normalizedPath;
+            if (!Utility::normalizedSyncPath(deleteOp->path(), normalizedPath)) {
+                normalizedPath = deleteOp->path();
+                LOGW_SYNCPAL_WARN(_logger, L"Failed to normalize: " << Utility::formatSyncPath(deleteOp->path()));
+            }
             if (auto createFileOpSetIt = _createFileOperationSet.find(deleteOp->path());
```

Comment:
`normalizedPath` should be used here?
`normalizedPath` is set but never read

Reply:
Done here: [0739172](https://github.com/Infomaniak/desktop-kDrive/pull/417/commits/073917246e02cb0ed9d0880581c83e886e37094a)